includes/admin/class-wcmp-export-consignments.php

```diff
@@ -471,4 +471,35 @@ private function setBaseData(): void
             ->setDeliveryType($this->getDeliveryType())
             ->setLabelDescription($this->getFormattedLabelDescription());
     }
+
+    /**
+     * @throws Exception
+     */
+    public function validate(): bool
+    {
+        if (! $this->consignment->validate()) {
+            return false;
+        }
+
+        $weight         = $this->getTotalWeight(); // NOTE this is already the weight per consignment (in grammes)
+        $maxColloWeight = WCMP_Data::MAX_COLLO_WEIGHT_PER_PACKAGE_TYPE[$this->getPackageType()];
+        if (! $maxColloWeight) {
+            $maxColloWeight = WCMP_Data::MAX_COLLO_WEIGHT_PER_PACKAGE_DEFAULT;
+        }
+
+        if ($weight > $maxColloWeight) {
+            $translationKey = (1 === $this->orderSettings->getColloAmount())
```

Comment:
Hoewel ik het ook een beetje stom vind is er misschien wel ergens een constante hiervoor, iets in de richting van `DEFAULT_COLLO_AMOUNT` denk ik

Reply:
Nja ik heb de echte hoeveelheid colli nodig daar van de order, niet een constante toch?